ovos_config/__main__.py

```diff
@@ -39,30 +39,34 @@ def dictDepth(dic: dict, level: int = 1) -> int:
     return max(dictDepth(dic[key], level + 1)
                for key in dic)
 
+def walkDict(dic: dict, key: str, only_endpoints: bool = False):
+    yield from _walkDict(dic, key.lstrip("/").split("/"), key.startswith("/"), only_endpoints)
```

Comment:
If we would remove only one leading slash, the searching behavior would become more consistent:
Current implementation:
`//coordinate` would match `coordinate` in the toplevel folder. Its behavior is identical to searching for `/coordinate`.
Removing only one slash:
`//coordinate` would match any key matching `coordinate` one level below top-level.

Reply:
fixed to remove only one leading slash